composer.json

```diff
@@ -10,12 +10,12 @@
     ],
     "require": {
         "php": ">=5.4.0",
-        "illuminate/filesystem": "~5.0",
-        "illuminate/support": "~5.0",
-        "symfony/finder": "2.6.*"
+        "illuminate/filesystem": "~5.0|~5.1",
```

Comment:
Also, that change in the version constraint was wrong anyway. `~5.0` already allows 5.1 and indeed any 5.x.